File: core/Core/Models/Card.cs
using System;
using System.Collections.Generic;
using DateOnly = global::System.DateOnly;

namespace Core.Models;

public partial class Card
{
    public int CardId { get; set; }

    public required int AccountId { get; set; }

    public required string CardNumber { get; set; }

    public required string Cvv { get; set; }

    public System.DateOnly? ExpiryDate { get; set; }

    public required string Status { get; set; }

    public Account? Account { get; set; }
}

File: analytic/domain/card/card.py
from datetime import datetime


class Card:
    def __init__(self,
                 card_id: int,
                 account_id: int,
                 card_number: str,
                 cvv: str,
                 expiry_date: datetime,
                 status: str):
        self.card_id = card_id
        self.account_id = account_id
        self.card_number = card_number
        self.cvv = cvv
        self.expiry_date = expiry_date
        self.status = status
